unity-renderer/Assets/UIComponents/Scripts/Components/Image/ImageComponentView.cs

```diff
@@ -0,0 +1,69 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public interface IImageComponentView
+{
+    /// <summary>
+    /// Fill the model and updates the image with this data.
+    /// </summary>
+    /// <param name="model">Data to configure the image.</param>
+    void Configure(ImageComponentModel model);
+
+    /// <summary>
+    /// Set an image.
+    /// </summary>
+    /// <param name="sprite">A sprite.</param>
+    void SetImage(Sprite sprite);
+
+    /// <summary>
+    /// Active or deactive the loading indicator.
+    /// </summary>
+    /// <param name="isVisible">True for showing the loading indicator and hidding the image.</param>
+    void SetLoadingIndicatorVisible(bool isVisible);
+}
+
+public class ImageComponentView : BaseComponentView, IImageComponentView
+{
+    [Header("Prefab References")]
+    [SerializeField] internal Image image;
+    [SerializeField] internal GameObject loadingIndicator;
+
+    [Header("Configuration")]
+    [SerializeField] internal ImageComponentModel model;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        Configure(model);
+    }
+
+    public void Configure(ImageComponentModel model)
+    {
+        this.model = model;
+        RefreshControl();
+    }
+
+    public override void RefreshControl()
+    {
+        if (model == null)
+            return;
+
+        SetImage(model.sprite);
+    }
+
+    public void SetImage(Sprite sprite)
```

Comment:
Remember that when the final image is set, it should appear with a fade-in effect. This is planned for the future?

Reply:
Well, currently the `Image` UI component has the methods `SetImage(Sprite)` and a `SetLoadingIndicatorVisible(bool)`. For now, to load an image through this component, we can activate its loading spinner, deactivates it when the loading ends and call to the `SetImage()` method to show the image. We don't have specified in any place this behaviour of showing the image with a fade-in, but if finally it should be appears in this way, we will not have any problem to modify the `ImageComponentView` -> `SetImage()` to show the image in this way 👍 For now, if you don't mind, I prefer to wait for the "refinament" stage (surely with Romi) to make these kind of changes in the UI components.